data-kernel/src/components/landing-page.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+const LandingPage = () => {
+  // navigation link to countries export page
+  let route = useNavigate();
+  const routeChange = () => {
+    route("/country-export");
+  };
+
+  return (
+    <div className="container-fluid">
+      <div className="row">
+        <div className="col-md-12">
+          <h1 className="text-center text-primary mt-5 pt-5 pb-5">
+            Data Kernel_
+          </h1>
+          <p className="text-center lead pb-5 mb-5">
+            Welcome to the Data Kernel App
+          </p>
+          <div className="d-flex flex-row p-5 justify-content-center">
+            <div
+              role="button"
+              className="border rounded bg-primary text-white m-2"
+              onClick={routeChange}
+              style={{
```

Comment:
Just wondering, are you going to stylise your divs in JS or will you move it to a css file?

Reply:
For now, keeping styles in-line for the Minimum Viable Product, the further enhancement would be creating custom Sass variables and theme. This wouldn't fall into the scope of this MVP project.